src/main/java/ru/itis/security/filters/JwtAuthenticationFilter.java

```diff
@@ -1,14 +1,24 @@
 package ru.itis.security.filters;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.core.context.SecurityContextHolder;
-import ru.itis.security.auth.TokenAuthentication;
+import org.springframework.stereotype.Component;
+import ru.itis.security.auth.JwtAuthentication;
+import ru.itis.security.providers.JwtAuthenticationProvider;
 
 import javax.servlet.*;
 import javax.servlet.http.HttpServletRequest;
 import java.io.IOException;
 
-public class TokenAuthenticationFilter implements Filter {
+@Component
+public class JwtAuthenticationFilter implements Filter {
     private final static String AUTH_HEADER = "Authorization";
```

Comment:
You still don't need this variable. Inline it.

Reply:
I do need it